library/network/src/modules/SuSEFirewall.rb

```diff
@@ -154,9 +154,64 @@ def self.create
   # ----------------------------------------------------------------------------
   # SuSEFirewalld Class. Trying to provide relevent pieces of SF2 functionality via
   # firewalld.
-  class SuSEFirewalld < Module
+  class SuSEFirewalld < Firewall
+    attr_reader :special_all_interface_zone
+
+    # Valid attributes for firewalld zones
+    @@zone_attributes=[:interfaces, :masquerade, :modified, :ports, :protocols, :services]
+    # {enable,start}_firewall are "inherited" from SF2 so we can't use symbols
+    # there without having to change all the SF2 callers.
```

Comment:
Indeed but I thought we should take advantage of using symbols in hashes (less memory, faster comparisions). But I will change them to Strings